### ISSUE TITLE
[ANNOUNCEMENT] Project Migration

### ISSUE THREAD

**Issue by FrederickGeek8** [WilliamLCobb/iNDS#252]
_TLDR: A fork has been created that has merged changes for iPhone X support and fixed Dropbox Syncing https://github.com/iNDS-Team/iNDS_

Hello everybody!

I am relatively new to contributing to the iNDS project, however, it is pretty clear that the project has become relatively stagnant as of late and the original developer is somewhat hard to contact. Therefore, I have taken it upon myself to attempt to breathe some new life into this project and create a new Github Organization to oversee the development of the iNDS codebase.

I am hoping to give people who are willing to contribute their time and effort to the repository permission to merge pull requests, thus bringing much-needed features and fixes that would otherwise not be available to everybody else.

I have tremendous respect for @WilliamLCobb – I bear no ill will. I do, however, believe that the community and this project are in need of help if it wishes to remain operational.

### FAQ
**Q: How does this affect me? Why should I care?**
A: Since this project has become stagnant, important issues and features such as the lack of iPhone X support and the broken Dropbox Sync feature have gone unfixed, simply because the original developer was not here to merge the fixes with the main repository. With the new Github Organization, I hope to have a team which will be able to not only fix upcoming issues but merge any pull requests that are submitted by other developers.

**Q: What about the original developer (@WilliamLCobb )**
A: I have invited him as an owner of the organization, so he has complete control over the organization as do I. If Will Cobb chooses to come back to the project, he can assume the position of owner once again.

**Q: What about the Cydia repo?**
A: I am currently working on setting up a Cydia repo that those with jailbroken devices can use to download iNDS. I, however, do not have a jailbroken device, so I will be looking for users to assist in testing the new repository. If you are interested in testing [please express your interest in this issue in the new repository.](https://github.com/iNDS-Team/iNDS/issues/3).

**Q: What about the iEmulator website?**
A: I will be reaching out to them shortly with the project migration news and hopefully their website will be able to accommodate this change.

**Comment by OscarXvita** [comment]
cool!

**Comment by Kirzuo** [comment]
How long before the dropbox sync update? That update alone would be very helpful!

**Comment by FrederickGeek8** [comment]
@Kirzuo The repository at https://github.com/iNDS-Team/iNDS already should have the Dropbox sync update and iPhone X update. Sorry if that was not made clear. If you are having any troubles with it, please [open a new issue over at the new repository.](https://github.com/iNDS-Team/iNDS/issues/new?template=Bug_report.md)

**Comment by Kirzuo** [comment]
@FrederickGeek8 Ahh I see, as I dont currently have a Jailbroken device, its not possible to install it yet?

**Comment by FrederickGeek8** [comment]
@Kirzuo You should be able to install on jailed devices, however, we do not have widely available prebuilt images at the moment. I created a Gitter chatroom for the new repo over at https://gitter.im/iNDS-Team/iNDS if you have any questions. [I just published a Github Release / created a git tab for version 1.7.0 ( the latest "stable" version ) of iNDS, which you can find here.](https://github.com/iNDS-Team/iNDS/releases/tag/v1.7.0)

For those you are maybe viewing this at a later time and wondering if the link is still valid or updated, there should be a chat button available in the README. Stable source code release will be available [in the "Releases" tab,](https://github.com/iNDS-Team/iNDS/releases) and all this information will be available on the repository soon. 

If anyone has any questions that are not pertaining to the Project Migration, you can always [open a new issue in the new repository.](https://github.com/iNDS-Team/iNDS/issues/new/choose)
